### ISSUE TITLE
Crash in dev environment

### ISSUE THREAD

**Issue by The-Fireplace** [Chicken-Bones/CodeChickenCore#43]
After pointing the mod to the correct directory when it is loading, I get the following crash:
http://paste.ubuntu.com/11270269/

**Comment by Chicken-Bones** [comment]
I've seen this a few times and I'm not quite sure what causes it. Is it possible that you have NEI added to your dev classpath and in the mods folder?

**Comment by The-Fireplace** [comment]
NEI and CCC(the dev and src jars of both) are in the forgeroot/eclipse/mods folder, and the dev jars of both are added to the build path.

**Comment by Chicken-Bones** [comment]
That's probably the issue. You need to make them compile-only (so they're not added to the runtime classpath)

**Comment by The-Fireplace** [comment]
And how exactly do I do that? I can't find anything on Google about how to.
I, for now, have removed it from the build path, and am just ignoring the "errors" in my NEI compat classes.

**Comment by Chicken-Bones** [comment]
I presume you're using eclipse, in which case, find the "order and export" tab and don't export nei

**Comment by The-Fireplace** [comment]
Alright, thanks!

**Comment by The-Fireplace** [comment]
Ok, I have re-added them to the build path, gone to the Order and Export tab, and unchecked the boxes to export NEI and CCC. It resulted in the crash again. So, I restarted Eclipse and tried again, with the same result. I then checked both of them, restarted Eclipse(One post I found about similar issues said Eclipse doesn't update those until it restarts), and tried running it again, with the same crash. Then, I tried leaving CCC checked and NEI unchecked, and vice versa, and still got the crash.

tl;dr: Tried how you said to do it, and no matter how I do it, it crashes again.

**Comment by Chicken-Bones** [comment]
Unfortunately I don't use eclipse, so I don't know how to specify it properly. Try unchecking both, and sending me the full fml log.

**Comment by The-Fireplace** [comment]
Here you go:
http://paste.ubuntu.com/11382954/

**Comment by Chicken-Bones** [comment]
On the second line, where it says the java classpath
C:\Users\Lance\Documents\unlogic-ii\eclipse\mods\NotEnoughItems-1.8-1.0.5.82-dev.jar
eclipse isn't removing them from the runtime classpath properly

**Comment by Chicken-Bones** [comment]
perhaps http://stackoverflow.com/questions/9282440/how-do-i-edit-the-runtime-classpath-in-eclipse

**Comment by Chicken-Bones** [comment]
alternatively (and I have no idea if this will work) try moving NEI to a place that's not the mods folder

**Comment by The-Fireplace** [comment]
The stack overflow answer looks good, I will try it when I get the chance. As for your alternative suggestion, which you aren't sure about, I have tried that before, it results in a crash.

**Comment by Chicken-Bones** [comment]
A different crash, out of curiosity, which one?

**Comment by The-Fireplace** [comment]
Ok, I tried what the stack overflow, that only allows you to add and remove projects, not the jars.

**Comment by Chicken-Bones** [comment]
I just booted up eclipse, and may have figured it out.
http://puu.sh/i2rjv/702657a6c4.png
Click debug configurations and go to the classpath tab
http://puu.sh/i2ret/e6fc15b33d.png
select the project, click edit
http://puu.sh/i2rhH/f7c304cc7a.png
check the Only include exported entries box

Then you can go to the Order and Export tab and check everything but
NEI/CCC/stuff in your mods folder
http://puu.sh/i2rlJ/37f77f91ce.png

hopefully that works

**Comment by Chicken-Bones** [comment]
A bit of fiddling it appears you may need to delete and re-add the project
to the debug classpath window, I'm not entirely sure, but you should be
able to find something in there

**Comment by The-Fireplace** [comment]
Deleting and re-adding the project let me fix it. I will let you know if I have any other issues. As for the other crash, which you wanted to know about, I will edit in a link to it when I get home.
